.golangci.yml

```diff
@@ -27,7 +27,6 @@ linters:
     - copyloopvar
     - ginkgolinter
     - goconst
-    - gocyclo
```

Comment:
Removed because it says the complexity of `(*ChiaDataLayerReconciler).Reconcile` is too high. Unclear at this moment in time how much simpler we can make that function with all the resources this reconciler needs to manage.